packages/api-client-core/src/InternalModelManager.ts

```diff
@@ -44,61 +45,78 @@ export const internalFindOneQuery = (apiIdentifier: string) => {
     `;
 };
 
-export const internalFindManyQuery = (apiIdentifier: string, isFirstQuery = false) => {
+export const internalFindFirstQuery = (apiIdentifier: string, options?: RecordData) => {
```

Comment:
Overall, I like (and prefer) this separation, but was encouraged to avoid it when I did this. I think this is cleaner, though, with each function serving one purpose.

I think that once this changes, we'll have to update some stuff in the main monorepo, including docs — but I believe that only matters once we update the app to point to a new version of the client.